presto-docs/src/main/sphinx/connector/blackhole.rst

```diff
@@ -79,3 +79,29 @@ table property (default value is equal to 16)::
       rows_per_page = 2000,
       field_length = 100
     );
+
```

Comment:
I would change first paragraph of this doc. Replace:

The Black Hole connector works like the ``/dev/null`` device on Unix-like
operating systems for data writing and like ``/dev/null`` or ``/dev/zero``
for data reading.

with:


The Black Hole connector is a testbed for performing various kinds of tests. Primarily it is designed for high performance testing of other components. Thus it works like the ``/dev/null`` device on Unix-like operating systems for data writing and like ``/dev/null``or ``/dev/zero`` for data reading. However it has also some other features that allows testing Presto in a more controlled manner.